### ISSUE TITLE
Price update delay

### ISSUE THREAD

**Issue by napalm69** [rocwang/grocer-issues#128]
With the supermarket weekly specials always rotating on Sunday midnight, the prices in the grocer app don't update until late Monday evening or sometimes even Tuesday. 
I often get caught out when shopping on a Monday with the wrong prices.
Are other people aware of this?

**Comment by rocwang** [comment]
Yes, that's sort of a known issue as Grocer update the catalogue around 8am once a day. So it might miss the price updates after that.

I'm working on a new version which would refresh the catalogue 3 times a day, aiming to release in early April.
